scripts/015-sdl_psl1ght_libs.sh

```diff
@@ -2,7 +2,7 @@
 # sdl_psl1ght_libs.sh by Naomi Peori (naomi@peori.ca)
 
 ## Download the source code.
-wget --no-check-certificate https://github.com/zeldin/SDL_PSL1GHT_Libs/tarball/master -O sdl_psl1ght_libs.tar.gz
+wget --no-check-certificate https://github.com/sergiou87/SDL_PSL1GHT_Libs/tarball/master -O sdl_psl1ght_libs.tar.gz
```

Comment:
This is pointing to my repo, but I guess I'll change it back to yours if you're happy merging https://github.com/zeldin/SDL_PSL1GHT_Libs/pull/3

Reply:
I think it is fine for now, but you should make a note to come back and change it if @zeldin can merge that. The PR on his repo seems ok to me but I do not have permissions there.